experimental/packages/opentelemetry-sdk-node/test/util/resource-assertions.ts

```diff
@@ -317,14 +322,17 @@ const assertHasOneLabel = (prefix: string, resource: Resource): void => {
 
   assert.ok(
     hasOne,
-    'Resource must have one of the following attributes: ' +
+    'Must have one node Resource(s) starting with [' +
+      prefix +
```

Comment:
Added extra context to the test to aid in debugging, as this test was failing after my change -- identified and fixed the issue. But found the additional context was useful -- so left it here.